ansys_optical_automation/application/radiance_from_view.py

```diff
@@ -0,0 +1,38 @@
+# Python Script, API Version = V21
+import math
+
+# import sys
+# repo_path = r"your_repo_path"
+# sys.path.append(repo_path)
+from ansys_optical_automation.speos_process.speos_sensors import RadianceSensor
+
+
+def main():
+    SpaceClaim.Api.V21.Command.Execute("FlyThroughCamera")
```

Comment:
I do not think using a specific version here is a good idea.

Reply:
Solution can be define self.Command = scdm_api.Command in the base.py and 
```suggestion
    Radiance = RadianceSensor("Radiance_from_View", SpeosSim, SpaceClaim)
    Radiance.Command.Execute("FlyThroughCamera")
```

Reply:
@gmoysan do you want to give it a try or shall i do it?

Reply:
@StefanThoene how can I get access to your changes (lines 1-6 for example) to test ? I believe your changes have not been pushed to the main, so I can't clone the main to access them.

Reply:
@gmoysan you can "pull" or type "git pull" at pycharm terminal to get the changes from Stefan